application.go

```diff
@@ -21,22 +21,22 @@ import (
 	"fmt"
 	"net/url"
 	"time"
+	"github.com/golang/glog"
```

Comment:
It would be nice to have more formatted logging. And I think most of the companies have their own logging standard. In this it would be really awesome to use the interface (as I see only `Infof` is needed) and provide glog as default implementation.
In this case everybody can pass his own implementation or just use default glog.
What do you think about it?